account_financial_report/report/abstract_report_xlsx.py

```diff
@@ -187,10 +187,13 @@ def write_line(self, line_object):
         """Write a line on current line using all defined columns field name.
         Columns are defined with `_get_report_columns` method.
         """
+        lang = self.env.user.lang
+        format_date = self.env['ir.qweb.field.date'].with_context(
+            lang=lang).value_to_html
```

Comment:
Why do you use the `value_to_html()` instead of the `odoo.tools.format_date`?

Reply:
Because it's the first way that I found to change the format, I found it in https://github.com/odoo/enterprise/blob/3963662509db09208e42ab660cda83c93946cb11/sale_subscription/models/sale_order.py#L174.

`odoo.tools.format_date` seems cleaner and works just as well, thanks!